src/scripts/GameConstants.ts

```diff
@@ -710,6 +713,8 @@ namespace GameConstants {
         'Aerodactyl': 'Old Amber',
         'Lileep': 'Root Fossil',
         'Anorith': 'Claw Fossil',
+        'Cranidos': 'Skull Fossil',
+        'Shieldon': 'Armor Fossil',
```

Comment:
Can we swap these 2 around, just so it's in the same order as `FossilToPokemon`

Reply:
Done